tools/automation/cli_linter/rules/command_rules.py

```diff
@@ -0,0 +1,19 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from ..rule_decorators import command_rule
+
+
+@command_rule('Checking missing help for commands...')
+def missing_command_help_rule(linter, command_name):
+    if not linter.get_command_help(command_name):
+        print('--Command: `%s`- Missing help.' % command_name)
+        linter.mark_rule_failure()
+
+
+@command_rule('Checking for --ids parameter in list commands...')
+def no_ids_for_list_commands_rule(linter, command_name):
+    if command_name.split()[-1] == 'list' and 'ids' in linter.get_command_parameters(command_name):
```

Comment:
I would expect `command_name` here to be "list" automatically and not have to do this split.  Consider having `linter`, `command_name` and `command_path` for these rule types.

Reply:
The command_name (as a string) is used as keys to access lots of different information in the linter and will likely be used by almost every rule.
The command_path is not as useful except for identifying a particular aspect of a command (i.e. checking for list or update commands) which seems more of a situational use. I would also like to avoid duplication in the arguments given to the rules.